cmd/otelcol/config/collector/fargate_config.yaml

```diff
@@ -0,0 +1,168 @@
+# This collector config file is designed for use within an ECS task.
+# The collector should run in a sidecar container within an ECS task.
+config_sources:
+  env:
+    defaults:
+      METRICS_TO_EXCLUDE: []
+      IMAGES_TO_EXCLUDE: ["quay.io/signalfx/splunk-otel-collector"]
+
+extensions:
+  health_check:
+    endpoint: 0.0.0.0:13133
+  http_forwarder:
+    ingress:
+      endpoint: 0.0.0.0:6060
+    egress:
+      endpoint: "https://api.${SPLUNK_REALM}.signalfx.com"
+  zpages:
+    #endpoint: 0.0.0.0:55679
+  smartagent:
+    bundleDir: "${SPLUNK_BUNDLE_DIR}"
+    collectd:
+      configDir: "${SPLUNK_COLLECTD_DIR}"
+
+receivers:
+#  fluentforward:
+#    endpoint: 127.0.0.1:8006
+  jaeger:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:14250
+      thrift_http:
+        endpoint: 0.0.0.0:14268
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:55681
+  # This section is used to collect the OpenTelemetry Collector metrics
+  # Even if just a Splunk APM customer, these metrics are included
+  prometheus/internal:
+    config:
+      scrape_configs:
+        - job_name: 'otel-collector'
+          scrape_interval: 10s
+          static_configs:
+            - targets: ['0.0.0.0:8888']
+          metric_relabel_configs:
+            - source_labels: [__name__]
+              regex: '.*grpc_io.*'
+              action: drop
+  sapm:
```

Comment:
Same nit: we don't need sapm receiver, while sapm exporter is required.

Reply:
Removed sapm receiver in next commit.